index.md

```diff
@@ -16,98 +16,91 @@ order: 1
 
 * **Guidelines**  
 Different organizations and governments may require conformance with different accessibility standards, thus different tools support these standards. The W3C Web Content Accessibility Guidelines (WCAG) is internationally recognized as the standard for web accessibility, and supported by most tools.
 
 * **Language**  
-Evaluation tools support different languages. This includes the user interface of the tools, as well as the language of the content they support (for example for detecting language-related barriers).
+The support for different languages provided by evaluation tools. This includes the user interface of the tools, as well as the language of the content they support (for example for detecting language-related barriers).
 
 * **Type of Tool**  
-Lists the platform and type of tools that are available, e.g. authoring tool or browser plugin, command line, desktop or mobile application.
+Evaluation tools can be plugins (extensions) for authoring tools and web browsers, command line tools, desktop or mobile applications, and online services. Diffent tools may be combined, depending on your design and development process.
 
-* **Supported Formats**  (referred to as "Technology" in the filters)
-Lists the format the tool can test, e.g. HTML, CSS, PDF, etc.
+* **Supported Formats**
+Most evaluation tools check the accessibility of HTML content. Some also support other web technologies, such as WAI-ARIA, CSS, SVG, and PDF.
 
 * **Assists by**  
-Lists what the specific uses of the tool, e.g. generating reports, providing step-by-step guidance, or displaying information within a web page
+Some web accessibility evaluation tools can be used in more than one way. The following are some common tool characteristics used to support the evaluation process:
 
-* **Automatically Checks**  
-The scope of what the evaluation tool can test. Sometimes a single page. Other times entire groups of related pages.
+** *Reports*. Reports generally list issues identified on many pages or whole sites. They are useful to help developers focus their attention on potential conformance issues can be automatically checked.
 
-* **License**  
-Which licenses the evaluation tool offers, e.g. Open Source, Commercial, Enterprise, etc.
+** *Step-by-step evaluations*. Wizard-based evaluation tools guide users through sequences of checks, step by step. They are able to conduct automated checks and prompt the user to manually assess the rest. For example a wizard-based tool may check images for alternative text. The user then evaluates how appropriate the alternative text is.
 
-### Other Features
+** *In-page feedback*. These tools, usually online services, insert temporary icons and markup to display results of accessibility checks within the actual pages. They are useful to view issues in context. They are also useful to understand the relative importance of each issue.
 
-* **Authoring Tools**  
```

Comment:
Reorganizing content - I fail to see why that was done and feel strongly it should have been left the way it was.

Reply:
Same as above.